tests/version.rs

```diff
@@ -273,3 +273,59 @@ fn version_about_multi_subcmd_override() {
         false
     ));
 }
+
+#[test]
+fn disabled_version_long() {
+    let app = App::new("foo").setting(AppSettings::DisableVersionFlag);
+    assert!(utils::compare_output(
```

Comment:
I know other tests in this file use this testing method, but I'd like to get away from using it unless we're testing something like wording or format of the error message itself. Instead one should use:

```rust
    let res = app.try_get_matches_from("foo --version".split(" "));
    assert!(res.is_err());
    let err = res.unwrap_err();
    assert_eq!(err.kind, clap::ErrorKind::UnknownArgument);
    assert_eq!(err.info, ["--version"]);
```

This way the error won't start failing if we change the format of the message.

Reply:
Thanks, changed in 55998aace3fc6878cd93ed63b3a5a6d31196c0b3.